_includes/header.html

```diff
@@ -83,6 +83,7 @@
       <img
         src="{{ optimize }}{{ page.image }}"
         class="m-8 w-full rounded-xl shadow-xl aspect-video max-w-3xl"
+        alt="{{ include.title }}"
```

Comment:
Hey, there will be no `title` value passed via the include tag. This should be `page.title`.